### PR TITLE
Updated google opentelemetry python exporter registry

### PR DESCRIPTION
**Pull request by aabmass**
- The exporters are a single package now, so combined them into one
- Updated repo link, authors, and description
- Added a registry entry for the opentelemetry-tools-google-cloud package.
  * creates a new category "tools"

### CODE REVIEW

File: content/registry/tools-python-google-cloud.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Google Cloud Tools
+registryType: tools
```
Comment (aabmass) [inline content/registry/tools-python-google-cloud.md:3]: There might be a better category name for this than "tools". The package has a trace propagator and a resource detector. Thoughts?

File: assets/sass/registry.sass [hunk 2]
```diff
@@ -5,6 +5,7 @@ $dark-blue: #39a9da;
 $core: #2C3ED3
 $instrumentation: #D32C3E
 $exporter: #3ED32C
+$tools: #2CD3B4
```
Comment (aabmass) [inline assets/sass/registry.sass:8]: I just generated this color with some tool online. Is there a better color available?

File: content/registry/tools-python-google-cloud.md (hunk 1 above)
Comment (austinlparker) [reply content/registry/tools-python-google-cloud.md:3]: 'extensions', maybe?

File: assets/sass/registry.sass (hunk 2 above)
Comment (austinlparker) [reply assets/sass/registry.sass:8]: it's probably fine for now

File: content/registry/tools-python-google-cloud.md (hunk 1 above)
Comment (aabmass) [reply content/registry/tools-python-google-cloud.md:3]: I noticed the registry page says (https://opentelemetry.io/registry/)
>The OpenTelemetry Registry allows you to search for instrumentation libraries, tracer implementations, utilities, and other useful projects in the OpenTelemetry ecosystem.

Maybe "utilities" then?

Comment (austinlparker) [reply content/registry/tools-python-google-cloud.md:3]: yeah, let's go with that, it's probably the most accurate for now.

**Pull request merged**